packages/ui/src/Layouts/PageLayoutHeader.tsx

```diff
@@ -96,3 +96,11 @@ const HeaderLinksWrapper = styled.div`
   justify-content: center;
   display: flex;
 `
+const HeaderButtonsWrapper = styled.div`
+  display: flex;
+  align-items: center;
+
+  & button {
```

Comment:
Nitpick, no need for `&` because we also have empty space after it

Reply:
updated :)